README.md

```diff
@@ -251,8 +251,8 @@ If you want to use the ONNXInfer as the bandwidth estimator, you should specify
 
     ONNXInfer:
     ``` shell
-    sudo docker run -d --rm -v `pwd`/examples/peerconnection/serverless/corpus:/app -w /app --name alphartc alphartc peerconnection_serverless receiver.json
-    sudo docker exec alphartc peerconnection_serverless sender.json
+    sudo docker run -d --rm -v /home/wang/AlphaRTC/examples/peerconnection/serverless/corpus:/app -w /app --name="alpha" alphartc peerconnection_serverless sender_pyinfer.json
```

Comment:
Did you intend to modify the `PyInfer` part? (Note that lines 254-255 are about running `ONNXInfer`)

Reply:
Yes,I want to modify the PyInfer part. I haven't tried ONNXInfer, but the same modifications about the environment apply to ONNXInfer

Reply:
I agree. However, ONNXInfer uses different sender/receiver JSON scripts that include config on paths to ONNX model (See https://github.com/OpenNetLab/AlphaRTC/blob/main/examples/peerconnection/serverless/corpus/sender.json#L14). So using `sender_pyinfer.json` and `receiver_pyinfer.json` for running ONNXInfer simply won't work.

Could you modify `PyInfer` part instead of `ONNXInfer` part, as that is what your modification does?